element_facemap/facial_behavior_estimation.py

```diff
@@ -230,20 +233,29 @@ def make(self, key):
 
         if task_mode == 'trigger':
             from facemap.process import run as facemap_run
-            facemap_params, motSVD, movSVD = (FacemapTask & key).fetch1('facemap_params', 'do_mot_svd', 'do_mov_svd')
+            params = (FacemapTask & key).fetch1('facemap_params')
+
+            # motSVD is initially a boolean and then is overwritten with a combination
+            # of lists and arrays
+            if (params['motSVD'] is True) or (len(params['motSVD']) > 0):
+                do_motSVD = True
+            else:
+                do_motSVD = False
+
+            if (params['movSVD'] is True) or (len(params['movSVD']) > 0):
+                do_movSVD = True
+            else:
+                do_movSVD = False
```

Comment:
I think the above is duplicated. L244-248 could be trimmed. Would the following work for L240-243?
```suggestion
            # motSVD is initially a boolean and then is overwritten with a combination
            # of lists and arrays
            do_movSVD = True if params['motSVD'] else False # if True or non-empty
```

Reply:
`params['motSVD']` is not always a boolean.

Reply:
`if params['motSVD']` will return `False` if the result is an empty list, but the truth value of an `np.array` throws `DeprecationWarning: [...] ambiguous`. I suppose you could cover that case with
```suggestion
            # motSVD is initially a boolean and then is overwritten with a combination
            # of lists and arrays
            do_movSVD = (True if (params['motSVD'] or params['motSVD'].size > 0)
                         else False)
```

Reply:
This would definitely work: `do_movSVD = (True if (params['movSVD'] or params['movSVD'].size > 0)
                         else False)`

Reply:
Second thoughts! Not sure...

Reply:
This part is outdated after the most recent changes.